src/app/rundown-view/services/tv2-action-group.service.ts

```diff
@@ -14,6 +15,8 @@ export class Tv2ActionGroupService {
   private groupActionsByContentTypeReducer(grouping: Tv2ActionContentTypeGrouping, action: Tv2Action): Tv2ActionContentTypeGrouping {
     if (this.isCameraAction(action)) {
       grouping.camera.push(action)
+    } else if (this.isRemoteAction(action)) {
```

Comment:
Couldn't this just be a `switch`?
All these `isX` methods just looks at the `contentType` which all are a `Tv2ActionContentType`.

Reply:
Agreed, i'd prefer a switch over this.

Reply:
Changed to switch case with type assertions. Using type guards give greater flexibility in when we want to acknowledge an action as the specified type – e.g. if a certain metadata attribute is set correctly.